src/components/UIShared/LoadingPlaceholder.vue

```diff
@@ -156,6 +156,33 @@ export default {
 	}
 }
 
+// Event cards placeholder ruleset
+.placeholder-list--event-cards {
+	display: flex;
+	flex-wrap: nowrap;
```

Comment:
This still creates horizontal scroll, if items are still loading
```suggestion
	flex-wrap: nowrap;
	overflow: hidden;
```